### DISCUSSION TITLE
Validating Data with Dagster Type Factories

### DISCUSSION THREAD

**Discussion by JSteinbauer** [dagster-io/dagster#10333] [Ideas]
Since this seems to be a common use case: **Why isn't there any class method to create a DagsterType from a DataFrameSchema?**

The case of validating pandas dataframes is covered in detail in  [https://docs.dagster.io/guides/dagster/dagster_type_factories#setup](url). While reading through it, I was surprised by how tedious it is to create such a validation.

Best, 
Jakob Steinbauer

**Comment by sryza** [comment]
Hey @JSteinbauer - have you had a chance to look at https://docs.dagster.io/integrations/pandera?

**Reply by JSteinbauer** [reply]
I didn't know about this additional module - thx for the info!

**Reply by sryza** [reply]
We should link to it from that guide
